pyproject.toml

```diff
@@ -11,6 +11,7 @@ python-dotenv = "^0.14.0"
 requests = "^2.26.0"
 pytest = "^7.0.1"
 selenium = "^4.1.2"
+markupsafe = "2.0.1"
```

Comment:
If this is to fix the bug with the nested markupsafe dependency, the nicer fix is updating Flask to the latest version.